docs/admin/extensible-admission-controllers.md

```diff
@@ -23,35 +23,38 @@ This page describes how to use Initializers and External Admission Hooks.
 ## Initializers
 
 ### what are initializers?
-* muntating AC; called in serial
-* a plugin-style initializer admission controller sets the default list of initializers for objects in the CREATE request. The default list of initializers can be dynamically configured via `InitializersConfiguration`, see XXX
-* an `initializer agent` is the worker that mutates the objects. is a controller watch for objects with `metadata.finalizers[0]==<initializer id>`
 
-* uninitialized objects are not visible ...
+Two meanings:
+* A list of pending pre-initialization tasks, stored in every object's metadata (e.g., "AddMyCorporatePolicySidecar").
+* The controllers which actually perform those tasks. The name of the task corresponds to the controller which performs the task.
 
-### When are initializers called?
+Once the controller has performed its assigned task, it removes its name from the the list. For example, it may send a PATCH that inserts a container in a pod and also removes its name from `metadata.initalizers`. Initializers may make mutations to objects.
 
-TODO: Explain the plugin "Initializer" admission controller
+Objects which have a non-empty initializer list are considered uninitialized, and are not visible in the API unless specifically requested (`?includeUninitialized=true`).
+
+### How are initializers triggered?
+
+When an object is POSTed, it is checked against all existing `InitializerConfiguration` objects (explained below). For all that it matches, all `spec.initializers[].name`s are appended to the new object's `metadata.initializers` field.
 
 ### Enable initializers alpha feature
 
-Initializers is an alpha feature, which is disabled by default. To turn it on, you need to
+Initializers are an alpha feature, which is disabled by default. To turn it on, you need to:
 
-* include "Initializer" in the `--admission-control` flag when starting the apiserver. If you are using HA-master or aggregated apiservers, you need to do so for each apiserver.
+* Include "Initializer" in the `--admission-control` flag when starting `kube-apiserver`. If you have multiple `kube-apiserver` replicas, all should have the same flag setting.
```

Comment:
It's not just when starting `kube-apiserver`, if an extension apiserver wants to use initializer, they also need to put "Initializer" in the list.